src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -368,129 +369,30 @@ class EntityClient (requestContext: RequestContext, modelSchema: ModelSchema, go
 
   def importPFB(workspaceNamespace: String, workspaceName: String, pfbRequest: PfbImportRequest, userInfo: UserInfo): Future[PerRequestMessage] = {
 
-    // initial validation of the presigned url provided as an argument
-    def validatePfbUrl(pfbRequest: PfbImportRequest)(op: URL => Future[PerRequestMessage]) = {
-      val pfbUrl = try {
-        new URL(pfbRequest.url)
-      } catch {
-        case e: Exception => throw new FireCloudExceptionWithErrorReport(ErrorReport(BadRequest, s"Invalid URL: ${pfbRequest.url}", e))
-      }
-      val acceptableProtocols = Seq("https") //for when we inevitably change our mind and need to support others
-
-      if (!acceptableProtocols.contains(pfbUrl.getProtocol)) {
-        throw new FireCloudExceptionWithErrorReport(ErrorReport(BadRequest, "Invalid URL protocol: must be https only"))
-      }
+    def enc(str: String) = java.net.URLEncoder.encode(str, "utf-8")
 
-      // TODO: add real (semantic) validation for origins, to ensure caller isn't supplying malice
-      op(pfbUrl)
-    }
+    // the payload to Import Service sends "path" and filetype.  Here, we force-hardcode filetype because this API
+    // should only be used for PFBs.
+    val importServicePayload: ImportServiceRequest = ImportServiceRequest(path = pfbRequest.url.getOrElse(""), filetype = "pfb")
```

Comment:
It feels weird to submit an empty string for the url, knowing it will fail - but I also understand that you're pushing all validation down to import service

Reply:
I could make it an `Option` and omit sending the entire key ... would that be better?

Reply:
up to you - this is fine too!